🚀Added Image Processing

Lakhankumawat opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.
Help us understand your motivation by explaining why you decided to make this change.
Happy Contributing!
-->

# Description

Added Python image processing techniques uses and tools.

## Fixes #590

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/Lakhankumawat/Amazing-Python-Scripts/blob/Experimental/Image%20Processing/README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

Lakhankumawat commented:
Actually i had earlier PR #534 which was not merged although i created a branch but it had those commits so deleted earlier commits and then created branch please ignore those i haven't changed anything .

kaustubhgupta commented on Tictactoe-Using-Minmax/README.md:
```diff
@@ -25,3 +25,4 @@ No package needs to install
 
```
Remove this file

Lakhankumawat replied:
Actually as i mentioned in my earlier PR i modified this file but since it was not merged i revert the changes but fogot to remove a blank line that's why you are able to see a change please ignore that i haven't changed anything . 

**rest of the changes which are requested** - i will do that

kaustubhgupta reviewed (changes requested):
In all the scripts, take the input for image path

Lakhankumawat commented:
- [x]  Added Image path in reading images .

kaustubhgupta commented:
> In all the scripts, take the input for the image path

@Lakhankumawat  I meant to say: Take inputs from users about which image they want to convert, that corresponding image path

Lakhankumawat commented:
> > In all the scripts, take the input for the image path
> 
> @Lakhankumawat I meant to say: Take inputs from users about which image they want to convert, that corresponding image path

Done!!

antrikshmisri reviewed (changes requested):
@Lakhankumawat Why have you deleted README.md from [Tictactoe-Using-Minmax/
](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Tictactoe-Using-Minmax)

Lakhankumawat commented:
> @Lakhankumawat Why have you deleted README.md from [Tictactoe-Using-Minmax/ ](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Tictactoe-Using-Minmax)

Yeah I had clarified the issue  regarding this to @kaustubhgupta sir and i had clarified the query , although if you would see i haven't deleted it - @kaustubhgupta  sir may have done it .
> I have a copy of that README can i add it back .

Lakhankumawat commented:
Let me clear it once again from starting ...
I had this pull request #534 but due to some issues ,  although i hadn't copied it from anywhere it was labelled **plagiarism**
In this pr i had two games rock paper scissors and tic tac toe using GUI ( the one which is present currently in original repo is CLI and with cpu  / mine one was 2players GUI ) so instead of making new fOLDER i added those tic-tac-toe files in this [Tic-tac-toe using minmax](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Tictactoe-Using-Minmax) and for 
**README.md**  file i just added my code discription and details in his ReadMe file . 


When my PR #534 wasn't merged i reverted back changes and also changed [README.md](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Tictactoe-Using-Minmax) file so @kaustubhgupta  sir thought that i have added this file anonymously and then README was deleted .


I greatly greatly apologise for all issues caused especially @kaustubhgupta sir who is working hard to merge our spammy pull request , i know i annoyed him too much 😐
But this was only options i had

kaustubhgupta commented:
@Lakhankumawat Yes, you are right but you don't need to be sorry for anything. As this is a beginner-friendly program, you can make mistakes that's not an issue. I also made the mistake to delete the file and later realized that it was not required. I could have simply told you to revert the last commit. But anyways now you have pushed the file back and nothing will be affected. 

About that previous PR, I have re-evaluated the code and I think we can accept the code again. I would suggest making different issues for different games as it becomes easy for us to review plus it would be more scoring for you.

Lakhankumawat commented:
>thanks sir i will try to work on #534

antrikshmisri commented:
> Let me clear it once again from starting ...
> I had this pull request #534 but due to some issues , although i hadn't copied it from anywhere it was labelled **plagiarism**
> In this pr i had two games rock paper scissors and tic tac toe using GUI ( the one which is present currently in original repo is CLI and with cpu / mine one was 2players GUI ) so instead of making new fOLDER i added those tic-tac-toe files in this [Tic-tac-toe using minmax](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Tictactoe-Using-Minmax) and for
> **README.md** file i just added my code discription and details in his ReadMe file .
> 
> When my PR #534 wasn't merged i reverted back changes and also changed [README.md](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Tictactoe-Using-Minmax) file so @kaustubhgupta sir thought that i have added this file anonymously and then README was deleted .
> 
> I greatly greatly apologise for all issues caused especially @kaustubhgupta sir who is working hard to merge our spammy pull request , i know i annoyed him too much 😐
> But this was only options i had

@Lakhankumawat No worries , I just didn't know about it. There is nothing to apologize for everyone is bound to make mistakes ,even us.
